### DISCUSSION TITLE
Check past footage with new tag?

### DISCUSSION THREAD

**Discussion by sezbez** [blakeblackshear/frigate#5385] [General]
hi all - if i update my settings to look for cars - how do I make it apply to history of certain days? 

I am looking for footage for police but the time span is large and my frigate wasn't setup to tag cars (but it's recorded) it's just lots of footage to go through 

thank you

**Comment by NickM-27** [comment]
This isn't supported directly. You could download the footage as an MP4 and then apply that as a camera. https://docs.frigate.video/development/contributing#2-create-a-local-config-file-for-testing

**Comment by sezbez** [comment]
thanks that would work.
